src/reducers/subnetwork.test.js

```diff
@@ -0,0 +1,148 @@
+import {
+  ADD_SUPER_NODE,
+  LOAD_NETWORK,
+  NEW_NETWORK,
+  REMOVE_SUPER_NODE,
+} from 'actions';
+
+import reducer from './subnetwork';
+
+describe('Subnetwork Reducers', () => {
+  describe('DEFAULT', () => {
+    it('returns an empty array', () => {
+      expect(reducer(undefined, {})).toEqual([]);
+    });
+  });
+
+  describe('NEW_NETWORK', () => {
+    it('returns an empty array', () => {
+      expect(reducer(undefined, {
+        type: NEW_NETWORK,
+      })).toEqual([]);
+    });
+  });
+
+  describe('LOAD_NETWORK', () => {
+    it('returns subnetworks from action payload', () => {
+      const subnetworks = [1, 2, 3];
+      const action = {
+        type: LOAD_NETWORK,
+        payload: {
+          state: {
+            network: { subnetworks },
+          },
+        },
+      };
+
+      expect(reducer(undefined, action)).toEqual(subnetworks);
+    });
+
+    it('returns empty array if action payload has no subnetwork', () => {
+      const action = {
+        type: LOAD_NETWORK,
+        payload: {
+          state: {
+            network: {},
+          },
+        },
+      };
+
+      expect(reducer(undefined, action)).toEqual([]);
+    });
```

Comment:
Nice one, but can you use `describe` here? So the test will be clear.
- When action has a network with subnetworks (describe)
  - returns network subnetworks (it)
- When action has a network without subnetworks (describe)
  - returns an empty array (it)

😄